flink-connectors/flink-connector-elasticsearch5/src/main/resources/META-INF/NOTICE

```diff
@@ -1,201 +1,104 @@
-distribute, sublicense, and/or sell copies of the Software, and to
-permit persons to whom the Software is furnished to do so, subject to
-the following conditions:
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
 
-The above copyright notice and this permission notice shall be
-included in all copies or substantial portions of the Software.
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- com.carrotsearch:hppc:0.7.1
+- com.fasterxml.jackson.core:jackson-core:2.8.1
+- com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.8.1
+- com.fasterxml.jackson.dataformat:jackson-dataformat-smile:2.8.1
+- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.8.1
+- com.github.spullara.mustache.java:compiler:0.9.3
+- com.tdunning:t-digest:3.0
+- commons-codec:commons-codec:1.10
+- commons-logging:commons-logging:1.1.3
+- io.netty:netty:3.10.6.Final
+- io.netty:netty-buffer:4.1.6.Final
+- io.netty:netty-codec:4.1.6.Final
+- io.netty:netty-codec-http:4.1.6.Final
+- io.netty:netty-common:4.1.6.Final
+- io.netty:netty-handler:4.1.6.Final
+- io.netty:netty-resolver:4.1.6.Final
+- io.netty:netty-transport:4.1.6.Final
+- joda.time:joda-time:2.5
+- net.java.dev.jna:jna:4.2.2
+- org.apache.logging.log4j:log4j-api:2.7
+- org.apache.logging.log4j:log4j-to-slf4j:2.7
+- org.apache.lucene:lucene-core:6.3.0
+- org.apache.lucene:lucene-analyzers-common:6.3.0
+- org.apache.lucene:lucene-backward-codecs:6.3.0
+- org.apache.lucene:lucene-grouping:6.3.0
+- org.apache.lucene:lucene-highlighter:6.3.0
+- org.apache.lucene:lucene-join:6.3.0
+- org.apache.lucene:lucene-memory:6.3.0
+- org.apache.lucene:lucene-misc:6.3.0
+- org.apache.lucene:lucene-queries:6.3.0
+- org.apache.lucene:lucene-queryparser:6.3.0
+- org.apache.lucene:lucene-sandbox:6.3.0
+- org.apache.lucene:lucene-spatial:6.3.0
+- org.apache.lucene:lucene-spatial-extras:6.3.0
+- org.apache.lucene:lucene-spatial3d:6.3.0
+- org.apache.lucene:lucene-suggest:6.3.0
+- org.apache.httpcomponents:httpasynclcient:4.1.2
+- org.apache.httpcomponents:httpclient:4.5.3
+- org.apache.httpcomponents:httpcore:4.4.6
+- org.apache.httpcomponents:httpcore-nio:4.4.5
+- org.elasticsearch:elasticsearch:5.1.2
+- org.elasticsearch:securesm:1.1
+- org.elasticsearch.client:rest:5.1.2
+- org.elasticsearch.client:transport:5.1.2
+- org.elasticsearch.plugin:lang-mustache-client:5.1.2
+- org.elasticsearch.plugin:percolator-client:5.1.2
+- org.elasticsearch.plugin:reindex-client:5.1.2
+- org.elasticsearch.plugin:transport-netty3-client:5.1.2
+- org.elasticsearch.plugin:transport-netty4-client:5.1.2
+- org.yaml:snakeyaml:1.15
+
+This project bundles the following dependencies under the MIT license. (?)
+
+- net.sf.jopt-simple:jopt-simple:5.0.2
+
+This project bundles the following dependencies under the BSD License.
+See bundled licenses for details.
+
+- org.hdrhistogram:HdrHistogram:2.1.6
```

Comment:
Not sure whether this license should not be included under Public Domain per Creative Commons CC0.